### PR TITLE
Change service point function

### PR DESCRIPTION
**Pull request by TetianaParanich**

### CODE REVIEW

File: cypress.json [hunk 1]
```diff
@@ -16,4 +16,4 @@
     "grepFilterSpecs": true,
     "grepOmitFiltered": true
   }
-}
\ No newline at end of file
+}
```
Comment (dmitriyfolio) [inline cypress.json:19]: do you need this change?

File: cypress/integration/circulation-log/search.spec.js [hunk 2]
```diff
@@ -81,21 +86,23 @@ describe('ui-circulation-log', () => {
   });
 
   after('Delete all data', () => {
-    CheckinActions.createItemCheckinApi({
+    cy.wrap(CheckinActions.createItemCheckinApi({
```
Comment (dmitriyfolio) [inline cypress/integration/circulation-log/search.spec.js:89]: you don't need wrap in this place

File: cypress.json (hunk 1 above)
Comment (TetianaParanich) [reply cypress.json]: ,I deleted it

File: cypress/integration/circulation-log/search.spec.js (hunk 2 above)
Comment (TetianaParanich) [reply cypress/integration/circulation-log/search.spec.js]: I deleted it

**Review by IhorBohdan — APPROVED** [review]

**Pull request merged**